consensus/src/round_manager_test.rs

```diff
@@ -317,7 +324,7 @@ impl NodeSetup {
         let (round_manager_tx, _) = aptos_channel::new(QueueStyle::LIFO, 1, None);
 
         let mut local_config = local_consensus_config.clone();
-        local_config.enable_broadcast_vote(false);
+        local_config.enable_broadcast_vote(true);
```

Comment:
Isn't this the default? We should remove this if so. That way test can fail if someone tried to change the config.

Reply:
Yes. The default is true. Removed this statement.